homework/App.js

```diff
@@ -16,14 +16,26 @@ class App {
     // 1. Create the fixed HTML elements of your page
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
-    const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
-
+    this.root = document.getElementById('root');
+    this.header = Util.createAndAppend('header', this.root);
+    Util.createAndAppend('label', this.header, { text: 'HYF Repositories' });
+    this.select = Util.createAndAppend('select', this.header);
```

Comment:
`this.select` and `this.header` have no use for them outside of this method, So you may use a plain normal variable as `const` and it will work fine.